karma.conf.js

```diff
@@ -23,6 +23,7 @@ module.exports = function(config) {
       {pattern: 'src/**/*.js'},
       // ------------------------- Test Files ----------------------------------
       'test/*_test.js',
+      'test/**/*_test.js',
```

Comment:
This should just be testing the unit files, why are you including the integration tests too?

Reply:
Sorry, I adjusted it by moving our unit tests into their own folder and adjusting the matcher. I will have a lot of unit tests tied to just this event processor, so for organizational purposes it makes sense to have subfolders for unit testing, like so:

- unit
-- GoogleAnalyticsEventProcessor
--- (bunch of unit tests)
-- uuidv4
--- (a couple unit tests)
- integration